docs/SdkVersioning.md

```diff
@@ -0,0 +1,590 @@
+  commit id: "0.2.0-dev1"
+  commit id: "0.2.0-dev2" tag: "0.2.0-rc1"
+  branch release/v0.2.0
+  checkout main
+  commit id: "0.3.0-dev1"
+  checkout release/v0.2.0
+  cherry-pick id: "0.3.0-dev1" tag: "0.2.0-rc2 + 0.2.0"
+  checkout main
+  commit id: "0.3.0-dev2"
+  commit id: "0.3.0-dev3"
+  checkout release/v0.2.0
+  cherry-pick id: "0.3.0-dev3" tag: "0.2.1"
+  checkout main
+  commit id: "1.0.0-dev1" type: REVERSE
+  commit id: "1.0.0-dev2" tag: "1.0.0-rc1"
+  branch release/v1.0.0
+  checkout main
+  commit id: "1.1.0-dev1"
+  commit id: "1.1.0-dev2"
+  checkout release/v1.0.0
+  cherry-pick id: "1.1.0-dev2" tag: "1.0.0-rc2"
+  checkout main
+  commit id: "1.1.0-dev3"
+  commit id: "1.1.0-dev4"
+  checkout release/v1.0.0
+  cherry-pick id: "1.1.0-dev4" tag: "1.0.0-rc3 + 1.0.0"
+  checkout main
+  commit id: "1.1.0-dev5"
+  checkout release/v1.0.0
+  commit id: "1.0.1" tag: "1.0.1"
+```
+
+:warning: **Warning** :warning::
+
+Creating a new release branch before the stabilization of a (API) Release has been completed, has a risk of unwillingly introducing breaking changes too early.
+
+Instead, we propose to either:
+- Create a new release branch just before the breaking changes got introduced in the `main` branch
+- Or extend the lifetime of the last release before the breaking change.
+
+# Work Instructions
+
+This chapter contains a full walkthrough of the development- and release- workflows
+which are supported by the SdkVer, incl:
+
+- Development
+- Promotion:
+  - Development to Release
+  - Development to Release Candidate
+  - Release Candidate to Release
+- Hardening:
+  - Release Candidate
+  - Release
+
+> **NOTE**: This chapter provides a _simplified way_ of creating branches, commits, pull requests, etc. 
+>
+>The goal is **not** to specify the way-of-working, rather explain how the SdkVer works.
+
+## Preparing your environment
+
+First, create a commisery configuration file in the root of your GitHub repository:
+
+`.commisery.yml`:
+```yml
+# Enable SdkVer strategy
+version-scheme: "sdkver"
+```
+
+Next, install the [GitHub CLI](https://cli.github.com) for interacting with GitHub.
+
+
+## Development flow
+
+### Workflow
+Create a workflow for managing our development releases:
+
+`.github/workflows/dev.yml`:
+```yml
+name: Development Release
+on:
+  push:
+    branches:
+      - main
+      - 'release/v**'
+
+jobs:
+  commisery-action:
+    name: Create Development release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Development release
+        uses: tomtom-international/commisery-action@v2
+        with:
+          token: ${{ github.token }}
+          create-release: true     # Create a GitHub Release
+          version-prefix: v        # Adds the prefix `v` to the version (i.e. `v.0.1.0`)
+          sdkver-release-type: dev # Update the Development version
+```
+
+### Instructions
+
+First we should ensure that we are on the `main` branch, updated against `HEAD` revision.
```

Comment:
```suggestion
First we should ensure that we are on the `main` branch, updated against `HEAD` revision.
```

Is it worth to make clear we assume `main` is the default branch? As miles might vary for others in terms of their default branch.

Reply:
Added a note to the Environment-section